### PR TITLE
Update README.md task 5

### PR DESCRIPTION
**Pull request by jalas167**

### CODE REVIEW

File: tasks/5-best-provider-selection/README.md [hunk 1]
```diff
@@ -46,10 +46,10 @@ You could also keep a list of any benchmarking data about providers in a file fo
 
 ### Useful links to solve this task
 
-- https://docs.golem.network/creators/javascript/mid-level/examples/hello/
-- https://docs.golem.network/creators/javascript/mid-level/
-- https://docs.golem.network/creators/javascript/docs/
-- https://docs.golem.network/creators/javascript/docs/readme/
+- https://docs.golem.network/docs/creators/javascript/mid-level/examples/hello/
```
Comment (cryptobench) [inline tasks/5-best-provider-selection/README.md:49]: Broken link

File: tasks/5-best-provider-selection/README.md [hunk 2]
```diff
@@ -46,10 +46,10 @@ You could also keep a list of any benchmarking data about providers in a file fo
...
+- https://docs.golem.network/docs/creators/javascript/mid-level/
```
Comment (cryptobench) [inline tasks/5-best-provider-selection/README.md:50]: Broken link

File: tasks/5-best-provider-selection/README.md [hunk 3]
```diff
@@ -46,10 +46,10 @@ You could also keep a list of any benchmarking data about providers in a file fo
...
+- https://docs.golem.network/docs/creators/javascript/docs/
```
Comment (cryptobench) [inline tasks/5-best-provider-selection/README.md:51]: Broken link

File: tasks/5-best-provider-selection/README.md [hunk 4]
```diff
@@ -46,10 +46,10 @@ You could also keep a list of any benchmarking data about providers in a file fo
...
+- https://docs.golem.network/docs/creators/javascript/docs/readme/
```
Comment (cryptobench) [inline tasks/5-best-provider-selection/README.md:52]: Broken link

File: tasks/5-best-provider-selection/README.md [hunk 5]
```diff
@@ -91,8 +91,8 @@ can be found at **https://preview.golem.network/**
 
 ## Useful links:
 
-- Javascript docs: https://docs.golem.network/creators/javascript/quickstart/
-- Dapps docs: https://docs.golem.network/creators/dapps/ 
+- Javascript docs: https://docs.golem.network/docs/creators/javascript/quickstart/
```
Comment (cryptobench) [inline tasks/5-best-provider-selection/README.md:94]: Broken link, should be https://docs.golem.network/docs/creators/javascript/quickstarts/quickstart

**Review by cryptobench — CHANGES_REQUESTED** [review]

**Comment by jalas167** [comment]
will not patch, as mid-level api docs are no longer available, and removing the links would remove the context

**Pull request closed without merging**